src/api.js

```diff
@@ -229,6 +230,27 @@ class DICOMwebClient {
         }
       }
 
+      if ('abortSignal' in options && options.abortSignal) {
+        const { abortSignal } = options;
+
+        if(abortSignal.aborted) {
+          request.abort();
```

Comment:
Why would you cancel it here?  This feels like you can only cancel on initiation, and why bother even calling on initiation for that?

Reply:
This is just in case it is cancelled prior to initiation (for whatever crazy reason). If this is not checked then, for the edge case where it is cancelled prior to initiation, the request will still go through. It may feel like you can only cancel on initiation, but that feeling is wrong - the `abortCallback` handles the cancel during the request itselft.